symengine/expression.h

```diff
@@ -120,11 +120,22 @@ class Expression
     {
         return eq(*m_basic, *other.m_basic);
     }
+
+    bool operator<(const Expression &other) const
+    {
+        return eq(*m_basic, *other.m_basic);
+    }
+     
     //! Overload check not equal (!=)
     bool operator!=(const Expression &other) const
     {
         return not (*this == other);
     }
+    //! Overload (>)
+    bool operator>(const Expression &other) const
+    {
+        return other.m_basic->compare(*m_basic);
```

Comment:
Until there is mathematical comparison in SymEngine, we shouldn't overload these operators as users can get confused if the structural comparison is given for `<` and `>`.